openpype/hosts/max/plugins/publish/validate_viewport_camera.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.max.api.lib import get_current_renderer
+
+from pymxs import runtime as rt
+
+
+class ValidateViewportCamera(pyblish.api.InstancePlugin,
+                             OptionalPyblishPluginMixin):
+    """Validates Viewport Camera
+
+    Check if the renderable camera in scene used as viewport
+    camera for rendering
+    """
+
+    order = pyblish.api.ValidatorOrder
+    families = ["maxrender"]
+    hosts = ["max"]
+    label = "Viewport Camera"
+    optional = True
+    actions = [RepairAction]
+
+    def process(self, instance):
+        if not self.is_active(instance.data):
+            return
+        cameras_in_scene = [c for c in rt.Objects
+                            if rt.classOf(c) in rt.Camera.Classes]
+        if rt.viewport.getCamera() not in cameras_in_scene:
+            raise PublishValidationError(
+                "Cameras in Scene not used as viewport camera"
+            )
```

Comment:
Wording is a bit unclear? Did you mean something like this.

```suggestion
        if rt.viewport.getCamera() not in cameras_in_scene:
            raise PublishValidationError(
                "Viewport camera is not a scene object"
            )
```
And this report should contain information what the artist should do and what it actually means ;)